### DISCUSSION TITLE
Access ALL valid data for your property

### DISCUSSION THREAD

**Discussion by SimpleProgrammingAU** [amberelectric/public-api#127] [Ideas]
I am currently an import customer with no exports back to the grid. However, one day I would like to change that.

Access to the valid Amber feed-in tariff for my property would be invaluable to me in making an informed decision.

Can this data also be shared?

**Comment by madpilot** [comment]
Not via the API at the moment, but if you email info@amber.com.au we can send you a CSV.

If you want to see the FiT right now, there is a toggle on the settings page that will
show the prices in the app.

![image](https://user-images.githubusercontent.com/19585/209417159-38574530-3273-4bab-bc9d-6474e33659a2.png)

**Reply by SimpleProgrammingAU** [reply]
Thank you for such a quick response, Christmas Eve no less. I have toggled it on in the app.

It would be very useful to expose this data via the API for customers like myself to be able to do assessments of usage, retail price and FIT concurrently without the need for hands-on work by Amber staff. :)

**Reply by captainpete** [reply]
+1, trying to simulate outcomes for a battery purchase
